### PR TITLE
added egld with multi transfer scenario

### PR DESCRIPTION
**Pull request by ssd04**
## Reasoning behind the pull request
- Integration tests for multi nft transfer with EGLD

## Testing procedure
- N/A

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: integrationTests/chainSimulator/vm/egldMultiTransfer_test.go [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+	tx = issueNonFungibleTx(2, addrs[0].Bytes, nftTicker, baseIssuingCost)
+
+	txResult, err = cs.SendTxAndGenerateBlockTilTxIsExecuted(tx, maxNumOfBlockToGenerateWhenExecutingTx)
+	require.Nil(t, err)
+	require.NotNil(t, txResult)
+
+	errMessage := string(txResult.Logs.Events[0].Topics[1])
+	require.Equal(t, vm.ErrCouldNotCreateNewTokenIdentifier.Error(), errMessage)
+
+	require.Equal(t, "success", txResult.Status.String())
```
Comment (miiu96) [inline integrationTests/chainSimulator/vm/egldMultiTransfer_test.go:347]: Can you add 2 more scenarios? 
- try to transfer egld with ESDTNFTMultiTransfer and the egld balance is insufficient 
- try to do multiple EGLD transfers with ESDTNFTMultiTransfer

Comment (ssd04) [reply integrationTests/chainSimulator/vm/egldMultiTransfer_test.go:605]: added more scenarios

**Review by miiu96 — APPROVED** [review]

File: integrationTests/chainSimulator/vm/egldMultiTransfer_test.go [hunk 2]
```diff
@@ -0,0 +1,606 @@
...
+
+	egldValue, _ := big.NewInt(0).SetString(beforeBalanceStr0, 10)
+	egldValue = egldValue.Add(egldValue, big.NewInt(13))
+	tx = multiESDTNFTTransferWithEGLDTx(2, addrs[0].Bytes, addrs[1].Bytes, [][]byte{nftTokenID}, egldValue)
+
+	txResult, err = cs.SendTxAndGenerateBlockTilTxIsExecuted(tx, maxNumOfBlockToGenerateWhenExecutingTx)
+	require.Nil(t, err)
+	require.NotNil(t, txResult)
+
+	fmt.Println(txResult)
```
[the remark this answers is not in the document]
Comment (ssd04) [reply integrationTests/chainSimulator/vm/egldMultiTransfer_test.go]: removed

File: integrationTests/chainSimulator/vm/egldMultiTransfer_test.go [hunk 3]
```diff
@@ -0,0 +1,606 @@
...
+		Version:   1,
+		Signature: []byte("dummySig"),
+		ChainID:   []byte(configs.ChainID),
+	}
+
+	txResult, err = cs.SendTxAndGenerateBlockTilTxIsExecuted(tx, maxNumOfBlockToGenerateWhenExecutingTx)
+	require.Nil(t, err)
+	require.NotNil(t, txResult)
+
+	fmt.Println(txResult)
```
[the remark this answers is not in the document]
Comment (ssd04) [reply integrationTests/chainSimulator/vm/egldMultiTransfer_test.go]: removed

File: integrationTests/chainSimulator/vm/egldMultiTransfer_test.go [hunk 4]
```diff
@@ -0,0 +1,606 @@
...
+	fmt.Println(string(txResult.Logs.Events[0].Topics[0]))
+	fmt.Println(string(txResult.Logs.Events[0].Topics[1]))
+
+	require.NotEqual(t, "success", txResult.Status.String())
+
+	eventLog := string(txResult.Logs.Events[0].Topics[1])
+	require.Equal(t, "insufficient funds for token EGLD-000000", eventLog)
+}
```
[the remark this answers is not in the document]
Comment (ssd04) [reply integrationTests/chainSimulator/vm/egldMultiTransfer_test.go:313]: added additional checks

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by miiu96 — DISMISSED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**